src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import './GoodsList.scss';
+
+const goodsFromServer: string[] = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+export const GoodsList = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [isReverseSort, setReverseSort] = useState(false);
+  const [isAlphabetSort, setAlphabetSort] = useState(false);
+  const [isLengthSort, setLengthSort] = useState(false);
```

Comment:
Try to merge. Just sortBy
```suggestion
  const [isAlphabetSort, setAlphabetSort] = useState(false);
  const [isLengthSort, setLengthSort] = useState(false);
```